keyboards/electronlab/klor/config.h

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2024 ElectronLab
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+// ┌─────────────────────────────────────────────────┐
+// │ p i n   c o n f i g                             │
+// └─────────────────────────────────────────────────┘ 
+
+#define I2C1_SCL_PIN GP3
+#define I2C1_SDA_PIN GP2
+
+
+// ┌─────────────────────────────────────────────────┐
+// │ s p l i t   c o m m u n i c a t i o n           │
+// └─────────────────────────────────────────────────┘ 
+
+//#define SPLIT_USB_DETECT
+
+//Half Duplex communication
+//#define SERIAL_USART_TX_PIN GP1     // USART TX pin
+//
+//Full Duplex communication
+//#define SERIAL_USART_TX_PIN GP4     // USART TX pin
+//#define SERIAL_USART_RX_PIN GP1     // USART RX pin
+//#define SERIAL_USART_FULL_DUPLEX
+//#define SERIAL_USART_PIN_SWAP
+
+
+// ┌─────────────────────────────────────────────────┐
+// │ o l e d s                                       │
+// └─────────────────────────────────────────────────┘
+
+#ifdef OLED_ENABLE
+#    define OLED_DISPLAY_128X64
+#    define SPLIT_OLED_ENABLE
+#    define OLED_FONT_H  "./glcdfont.c"
+#endif
+
+
+// ┌─────────────────────────────────────────────────┐
+// │ a u d i o                                       │
+// └─────────────────────────────────────────────────┘
+
+#define AUDIO_PWM_DRIVER PWMD4
+#define AUDIO_PWM_CHANNEL RP2040_PWM_CHANNEL_B
+#define AUDIO_STATE_TIMER GPTD4
+#define AUDIO_PIN GP9
+#define AUDIO_DAC_SAMPLE_MAX 4095U
+#define AUDIO_VOICE_DEFAULT butts_fader
+#ifdef AUDIO_ENABLE     
+#    define KLOR_SOUND W__NOTE(_DS0), W__NOTE(_DS1), H__NOTE(_DS2), H__NOTE(_DS3), Q__NOTE(_DS4), Q__NOTE(_DS5), E__NOTE(_DS6), E__NOTE(_DS7), S__NOTE(_DS8), Q__NOTE(_GS0)
+#    define STARTUP_SONG SONG(KLOR_SOUND)
+#    define BYE_SOUND H__NOTE(_DS4), H__NOTE(_DS3), W__NOTE(_DS1)
+#    define GOODBYE_SONG SONG(BYE_SOUND)
```

Comment:
Just a heads up, you could also do this: 
```suggestion
#    define STARTUP_SONG SONG(W__NOTE(_DS0), W__NOTE(_DS1), H__NOTE(_DS2), H__NOTE(_DS3), Q__NOTE(_DS4), Q__NOTE(_DS5), E__NOTE(_DS6), E__NOTE(_DS7), S__NOTE(_DS8), Q__NOTE(_GS0))
#    define GOODBYE_SONG SONG(H__NOTE(_DS4), H__NOTE(_DS3), W__NOTE(_DS1))
```

Reply:
Agreed. Change is done and committed.